src/k8s/pkg/utils/certificate.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"net"
+	"strings"
+)
+
+func GetExtraSANsFromString(extraSANs string) []string {
+	// TODO: Add validation for the extraSANs
+	return strings.Split(extraSANs, ",")
+}
+
+func SeparateSANs(extraSANs []string) ([]net.IP, []string) {
```

Comment:
This is not a useful name for callers to understand what this funvtion does wothout having to look up the implementation. Maybe call this something like SplitIPAndDNSSANs.